providers/default/defltprov.c

```diff
@@ -111,13 +111,20 @@ static const OSSL_ALGORITHM deflt_ciphers[] = {
     { "AES-256-CTR", "default=yes", aes256ctr_functions },
     { "AES-192-CTR", "default=yes", aes192ctr_functions },
     { "AES-128-CTR", "default=yes", aes128ctr_functions },
+/* TODO(3.0) Add aliases when they are supported */
```

Comment:
Isn't an alias as easy as adding another line with a different name? Why do we need aliases?

Reply:
It's mostly for presentation purposes, giving things like `openssl list -cipher-algorithms` the possibility to present a bunch of names denoting the same algorithm.  I would say *also* that it's probably good if we have a consistent way of keeping track of the diverse names each algorithm can have.

Note that I have a proposal for aliases (or well, multiple names, as I don't see why there should be a "main name") in #8985

Reply:
Currently there would need to be at least 2 entries for every algorithm (Shortname and longname). I really dont like the shortnames that are 'id- ', so for these we will have at least 3 entries. The TODO is there to indicate that this needs to be resolved - not necessarily in this PR.

Reply:
The shortname and longname in the object database are mostly used to get both a lowercase and an uppercase name, which is resolved in the new namenumber map with case-insensitive string comparison.  Remains the few cases where we really have two different names.

Reply:
Anyway, I think a TODO will suffice here.